lua/starfall/libs_cl/render.lua

```diff
@@ -1517,24 +1558,8 @@ end
 -- @param number rot Rotation in degrees
 function render_library.drawTexturedRectRotated(x, y, w, h, rot)
 	if not renderdata.isRendering then SF.Throw("Not in rendering hook.", 2) end
-
-	local rad = math.rad(rot)
-	local cos, sin = math.cos(rad), math.sin(rad)
-	makeQuad(-w/2, -h/2, w, h)
-	local function rotateVector(vec)
-		-- These locals are needed because the next line requires the vector to be unmodified
-		local x = vec.x * cos - vec.y * sin + x
-		local y = vec.x * sin + vec.y * cos + y
-		vec.x = x
-		vec.y = y
-	end
-
-	rotateVector(quad_v1)
-	rotateVector(quad_v2)
-	rotateVector(quad_v3)
-	rotateVector(quad_v4)
-
-	render.DrawQuad(quad_v1, quad_v2, quad_v3, quad_v4, currentcolor)
+	vec_SetUnpacked(quad_pos, x, y, 0)
+	render_DrawQuadEasy(quad_pos, quad_normal, w, h, currentcolor, -90 - rot)
```

Comment:
Need to performance test and test that this still works the same. makeQuad might be able to be deleted as well if it's not used elsewhere.

Reply:
This is about 3 times faster and works identical.